src/CheckoutSdk/Payments/Links/IPaymentLinksClient.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System.Threading;
+using System.Threading.Tasks;
+
+namespace Checkout.Payments.Links
+{
+
```

Comment:
Some blank lines that can be removed. Possibly you're using VS, not sure if it applies the same formatting. Please confirm if you're applying the same code style.